pom.xml

```diff
@@ -102,4 +108,38 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.0</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>all</shadedClassifierName>
```

Comment:
Just to be clear to the reviewers, we now produce two jar files - the standard one which doesn't have guava shaded and relocated and then an "all" version which you can depend on using a classifier. We did this in order to fix a downstream bug which this ended up not fixing but thought it would be useful to include this new artifact for other uses cases. The longer term fix could be removing the Guava dependency entirely but that's a bigger piece of work.

Reply:
If we don't directly need this shading atm, I rather not have it I think it adds more confusion.

Reply:
Yes, we discussed and since we don't actually have something downstream that could use this let's remove it.

Reply:
Ok, I've removed it 👍